new(guides): VTEX Ads docs migration - part II

julia-rabello opened the pull request:
[EDU-14825](https://vtex-dev.atlassian.net/browse/EDU-14825)

#### Types of changes
- [x] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)


[EDU-14825]: https://vtex-dev.atlassian.net/browse/EDU-14825?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

julia-rabello committed:
new(guides): VTEX Ads docs migration

PedroAntunesCosta committed:
fix callout emojis

PedroAntunesCosta committed:
update audiences integration flow

emersonme commented on docs/guides/VTEX Ads/exporting-data-from-vtex-ads/exporting-aggregated-data-from-vtex-ads.md:
there's some `YYYY-DD-MM` date format in this file, it should be `YYYY-MM-DD`

emersonme commented on docs/localization/integrating-vtex-ads-with-external-marketplaces.md:
````diff
@@ -0,0 +1,246 @@
+- 24h expiration time
+
+```http
+GET https://app.newtail.com.br/login/marketplace?sso=JWT
````
domain should be app.ads.vtex.com

PedroAntunesCosta committed:
Merge branch 'main' into vtex-ads-docs-part-2

PedroAntunesCosta committed:
fix(vtex-ads): address PR review comments for docs migration part II

- Fix YYYY-DD-MM date format to YYYY-MM-DD in aggregated data tables
- Replace app.newtail.com.br with app.ads.vtex.com in SSO login URLs
- Fix CDN image paths pointing to wrong directory (guides → localization)
- Fix markdownlint: trailing spaces, blank lines in blockquotes, list
  indentation, blank lines around headings and fenced code blocks,
  and missing language specifiers on HTTP code blocks

PedroAntunesCosta committed:
Update integrating-audiences.md

PedroAntunesCosta committed:
content architecture

PedroAntunesCosta committed:
Merge branch 'main' into vtex-ads-docs-part-2
